config/gcpmanager/gcp-manager.yaml

```diff
@@ -19,7 +13,7 @@ spec:
       labels:
         app: gcp-manager
     spec:
-      serviceAccountName: gcp-manager
+      serviceAccountName: substratus
```

Comment:
Prefer to use "sci" for this serviceAccountName. This should not be the same as the controller manager. Keeping the controller manager free of cloud privileges is better from a security perspective.

Reply:
fixed